client/src/components/tests/LeftAndMain-story.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf, setAddon } from '@storybook/react';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs/react';
+import { withNotes } from '@storybook/addon-notes';
+import JSXAddon from 'storybook-addon-jsx';
+import notes from '../README.md';
+
+import LeftAndMain from '../LeftAndMain';
+
+
+setAddon(JSXAddon);
+
+storiesOf('ReactAdmin/LeftAndMain', module)
+  .addDecorator(story => <div>{story()}</div>)
+  .addDecorator(withKnobs)
+  .addWithJSX(
+    'LeftAndMain',
+    withNotes(notes)(() => (
+      <LeftAndMain fakeProp={text('Fake Property', 'Hello world')}>
```

Comment:
The story has a "Knobs" tab. Methods like `text()`, `boolean()` and `select()` allow you to add form field to this tab that can be use to update props to your components on the fly.